JMS/com/ibm/mq/samples/jms/JmsPut.java

```diff
@@ -68,10 +69,18 @@ public class JmsPut {
     private static String CIPHER_SUITE;
     private static String CCDTURL;
     private static Boolean BINDINGS = false;
-
+    private static JwtHelper jh = null;
+    private static String accessToken = null;
     public static void main(String[] args) {
         initialiseLogging();
-        mqConnectionVariables();
+        SampleEnvSetter env = new SampleEnvSetter();
+        jh = new JwtHelper(env);
+        if (jh.isJwtEnabled()) {
+            accessToken = jh.obtainToken();
+        } else {
+            logger.info("One or more JWT Credentials missing! Expected APP_USER and APP_PASSWORD in MQ Endpoints");

```

Comment:
It is now an info message - good, but the second clause is alarming. It should be more like "One of more JWT Credentials missing! Will not be using JWT for authentication"